config/environment.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = function(environment) {
+  var results = {
+    APP: {
```

Comment:
I do not believe that you can mutate the second argument (if you can that is a bug that we should fix).